enterprise/server/remote_execution/containers/firecracker/firecracker_test.go

```diff
@@ -707,16 +712,15 @@ func TestFirecracker_RemoteSnapshotSharing_ManualBenchmarking(t *testing.T) {
 		t.Skip()
```

Comment:
nit - I missed this in the previous PR, but re. the comment above this test - when doing benchmarking we shouldn't enable race detection because it will slow down the test by a lot and give unrealistic numbers